modules/appeals_api/lib/appeals_api/health_checker.rb

```diff
@@ -48,7 +48,7 @@ def caseflow_is_healthy?
     def central_mail_is_healthy?
       return @central_mail_healthy unless @central_mail_healthy.nil?
 
-      @central_mail_healthy = !CentralMail::Service.current_breaker_outage?
+      @central_mail_healthy = CentralMail::Service.service_is_up?
```

Comment:
Smol, but lovely change in logic. I find the non-negated version easier to read.